Source/GoBot/GoBot.swift

```diff
@@ -263,7 +263,7 @@ class GoBot: Bot {
             }
             self.preloadedPubService?.preloadPubs(in: self, from: nil)
             
-            Task.detached(priority: .utility) {
+            Task.detached(priority: .background) {
```

Comment:
Since banning runs more queries now I bumped it down to .background QoS. This is nice because apple doesn't schedule the thread now until a few seconds after the app has launched, so it isn't hitting the db immediately while we are still trying to load the home feed and stuff.